BaseClasses.py

```diff
@@ -264,6 +264,7 @@ def set_item_links(self):
                         "local_items": set(item_link.get("local_items", [])),
                         "non_local_items": set(item_link.get("non_local_items", [])),
                         "link_replacement": replacement_prio.index(item_link["link_replacement"]),
+                        "skip_if_solo": item_link.get("skip_if_solo", False)
```

Comment:
```suggestion
                        "skip_if_solo": item_link.get("skip_if_solo", False),
```
Makes the diff better when something gets added later to have that comma on the end.
If the comma was not at the end of the line above this one, it would show both lines changed.